subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/repositories/resolver/IvyResourcePattern.java

```diff
@@ -51,10 +52,32 @@ public ExternalResourceName toVersionListPattern(ModuleIdentifier module, IvyArt
     }
 
     public ExternalResourceName toModulePath(ModuleIdentifier module) {
-        throw new UnsupportedOperationException("not implemented yet.");
```

Comment:
This new implementation is incorrect: it should return the _module_ path (that contains all the version directories). However, since this isn't required for an Ivy resolver, we can just leave it with `UnsupportedOperationException`.

Reply:
fixed